pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -315,6 +315,10 @@ const (
 	// Note that this annotation is alpha and can be removed anytime without further notice. Only use it if you know
 	// what you do.
 	ShootAlphaControlPlaneHAVPN = "alpha.control-plane.shoot.gardener.cloud/high-availability-vpn"
+	// ShootAlphaControlPlaneDisableNewVPN is a constant for an annotation on the Shoot resource to disabling the
+	// new Go implementation of VPN.
+	// TODO (MartinWeindel) Remove after experience shows, that new VPN implementation is working smoothly.
```

Comment:
```suggestion
	// TODO(MartinWeindel) Remove after experience shows, that new VPN implementation is working smoothly.
```

Doesn't it make more sense to only remove it when the feature gate gets promoted to GA? Otherwise, people would probably like to have the chance to opt out of it in case it is not stable.